src/registry_schemas/schemas/business.json

```diff
@@ -133,6 +152,29 @@
           "default": "",
           "examples": ["123456789"],
           "pattern": "^[0-9]{9}$"
+        },
+        "state": {
+          "type": "string",
+          "title": "The state of the business",
+          "enum": [
+            "ACTIVE",
+            "HISTORICAL",
+            "LIQUIDATION"
+         ]
+        },
+        "stateFiling": {
+          "type": "string",
```

Comment:
Should this be an enum, for example something from here?
https://github.com/bcgov/business-filings-ui/blob/main/src/enums/dissolutionTypesNames.ts

Reply:
The idea is to return the filing link that caused this state. Once you have that filing any info can be retrieved from that

Reply:
Sorry, this comment got attached to the wrong line of code (I added it while you pushed up a new version).

This comment is now obsolete.